i18n/en.json

```diff
@@ -875,6 +881,10 @@
   "admin.permissions.permission.manage_team.name": "Manage team",
   "admin.permissions.permission.manage_webhooks.description": "Create, edit, and delete incoming and outgoing webhooks.",
   "admin.permissions.permission.manage_webhooks.name": "Manage Webhooks",
+  "admin.permissions.permission.manage_incoming_webhooks.description": "Create, edit, and delete incoming webhooks.",
```

Comment:
Do the old translation strings also need to be removed from here, or will that get done automatically now?

Reply:
Can be done automatically using the tool, but must be committed.